web-animations-2/Overview.bs

```diff
@@ -800,6 +802,20 @@ The procedure to [=pause an animation=] needs to refer not only to the
 <ins cite="#amend-CE">Likewise, the procedure to [=pause an animation=] needs to include scheduling
 a task for updating [=custom effects=].</ins>
 
+Update the constraints for setting the seek time to only apply when using a
+monotonic timeline. A [=scroll-driven animation=] needs to defer setting the
+hold time until the animation range has been computed.
```

Comment:
I think this needs to be more specific. For example, you probably mean to update step 14 of the procedure to pause an animation https://drafts.csswg.org/web-animations-1/#pause-an-animation such that we have a pending pause task until this additional constraint is satisfied.